.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

Comment:
We don't currently pin our actions to specific commit hashes. So I'm not sure how much value dependabot will be?

Also, I'm not totally opposed to adding this but I find dependabot really annoying – even if only monthly. Maybe we can find a compromise, e.g. use the same schedule as for our pre-commit which is quarterly.

Reply:
It will bump from major to major.